tasks/README.md

```diff
@@ -38,12 +28,9 @@ To set up authentication, please refer to our
 To create a queue using the Cloud SDK, use the following gcloud command:
 
 ```
-gcloud beta tasks queues create-app-engine-queue my-appengine-queue
+gcloud beta tasks queues create <QUEUE_NAME>
```

Comment:
Can we provide an example queue name such that a developer can just copy/paste this command?